damus/Views/Settings/NotificationSettingsView.swift

```diff
@@ -175,6 +177,8 @@ struct NotificationSettingsView: View {
                     .toggleStyle(.switch)
                 Toggle(NSLocalizedString("Show only from users you follow", comment: "Setting to Show notifications only associated to users your follow"), isOn: self.notification_preference_binding($settings.notification_only_from_following))
                     .toggleStyle(.switch)
+                Toggle(hellthread_notification_settings_text, isOn: $settings.hellthread_notification)
```

Comment:
@tyiu, while testing the notepush changes (https://github.com/damus-io/notepush/pull/16#pullrequestreview-2741552143), I noticed an issue where toggling this does not sync up the setting with the server.

We should use the `notification_preference_binding` function to generate a binding that will automatically sync this setting with the server.

```suggestion
                Toggle(hellthread_notification_settings_text, isOn: self.notification_preference_binding($settings.hellthread_notification))
```

Reply:
Thanks! Fixed.